README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.5+](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@tambry 
hmm, I dunno, those who clicks the link do not really see the version before so much to say that the it will not download the old version is useless, so it's good like that ^^